packages/js/k6/src/frontend/scenarios.ts

```diff
@@ -16,59 +15,57 @@ const localePrefix = (locale: string) => {
   return locale === "en" ? "" : locale + "/"
 }
 
-const visitUrl = (url: string, action: Action) => {
-  // eslint-disable-next-line import/no-named-as-default-member
-  const response = http.get(url, {
-    headers: { "User-Agent": "OpenverseLoadTesting" },
-  })
-  const checkResponseFailed = makeResponseFailedCheck("", url)
-  if (checkResponseFailed(response, action)) {
-    console.error(`Failed URL: ${url}`)
-    return 0
-  }
-  return 1
-}
-
 const parseEnvLocales = (locales: string) => {
   return locales ? locales.split(",") : ["en"]
 }
 
-export function visitStaticPages() {
+export async function visitStaticPages() {
   const locales = parseEnvLocales(__ENV.LOCALES)
-  console.log(
-    `VU: ${exec.vu.idInTest}  -  ITER: ${exec.vu.iterationInInstance}`
-  )
+  const ovGroup = `visit static pages for locales ${locales}`
+
   for (const locale of locales) {
-    group(`visit static pages for locale ${locale}`, () => {
-      for (const page of STATIC_PAGES) {
-        visitUrl(
-          `${FRONTEND_URL}${localePrefix(locale)}${page}`,
-          "visitStaticPages"
+    for (const page of STATIC_PAGES) {
+      const url = new URL(`${FRONTEND_URL}${localePrefix(locale)}${page}`)
+      const response = await http.get(url, null, { tags: { ovGroup } })
+      const result = check(
+        response,
+        { "status was 200": (r) => r.status === 200 },
+        { ovGroup }
+      )
+
+      if (!result) {
+        console.error(
+          `Request failed ⨯ ${url}: ${response.status}\n${response.body}`
         )
       }
-    })
+    }
   }
 }
 
-export function visitSearchPages() {
+export async function visitSearchPages() {
   const locales = parseEnvLocales(__ENV.LOCALES)
-  const params = __ENV.PARAMS
```

Comment:
todo: add back the params functionality for the "search with filters" scenarios that use them.